Required to make Sink build.
The function "printStacktrace" makes use of backtrace which is not available on Alpine.
https://invent.kde.org/pim/sink/-/issues/2

Building of the examples and tests are broken
https://invent.kde.org/pim/sink/-/issues/1
diff --git a/CMakeLists.txt b/CMakeLists.txt
index 178b0293..9d320f40 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -150,11 +150,15 @@ add_subdirectory(common)
 # the synchronizer
 add_subdirectory(synchronizer)
 
-# example implementations
-add_subdirectory(examples)
+if(BUILD_EXAMPLES)
+	# example implementations
+	add_subdirectory(examples)
+endif()
 
-# some tests
-add_subdirectory(tests)
+if(BUILD_TESTS)
+	# some tests
+	add_subdirectory(tests)
+endif()
 
 # cli
 add_subdirectory(sinksh)
diff --git a/synchronizer/backtrace.cpp b/synchronizer/backtrace.cpp
index bd993ccb..597694bf 100644
--- a/synchronizer/backtrace.cpp
+++ b/synchronizer/backtrace.cpp
@@ -250,7 +250,9 @@ static void crashHandler(int signal)
         std::cerr << "Unexpected signal " << signal << " received\n";
     }
 
+#if defined(__GLIBC__)
     printStacktrace();
+#endif // __GLIBC__
 
     //Get the word out that we're going down
     if (sListener) {
